custom-words.txt

```diff
@@ -967,6 +967,8 @@ HSLA
 HSMs
 Hubspot
 Hugepages
+hybridcloud
+hybridcloudclient
```

Comment:
do you need to add this? Seems like a symptom of not properly capitalizing: hybridCloudClient, hybridCloud, HybridCloudClient etc.